### ISSUE TITLE
Connection fails to SQL Server instance on Windows

### ISSUE THREAD

**Issue by Alwigi** [prisma/prisma#4485]
## Bug description

When running npx prisma introspect I get the error below on a locally installed sql server instance:

    Error: Error in connector: Error querying the database: Error querying the database: Error querying the database: Conversion 
    error: Could not resolve SQL browser instance SQLSERVER2017STD

.env looks like this: 

    DATABASE_URL="sqlserver://MyPc\SQLSERVER2017STD:1433;initial catalog=MyDb;user=sa;password=1234"

schema.prisma looks like this:

    datasource db {
      provider = "sqlserver"
      url      = env("DATABASE_URL")
    }

    generator client {
      provider = "prisma-client-js"
      previewFeatures = ["microsoftSqlServer"]
    }

I installed prisma using:

    npm install @prisma/cli --save-dev

SQL server is enabled for TCP/IP on port 1433 in the config manager, and I can connect just fine using the same details with sql management studio (in TCP/IP mode).
The SQL browser service is running.
I have tried with no firewalls enabled although I doubt this would have any effect locally.
I also tried MyPc\\\SQLSERVER2017STD in DATABASE_URL but that gives 'invalid instance name' error so I assume that single \ is correct.

## How to reproduce

run 'npx prisma introspect' with the config as above

## Expected behavior

'npx prisma introspect' should execute and connect to sql server as expected.

## Prisma information

npx prisma introspect

## Environment & setup

- OS: Windows 10 Pro (10.0.19042 Build 19042)
- Database: SQL Server 2017 Standard
- Node.js version: v12.15.0
- Prisma version: 

```
@prisma/cli          : 2.12.1
@prisma/client       : Not found
Current platform     : windows
Query Engine         : query-engine cf0680a1bfe8d5e743dc659cc7f08009f9587d58 (at node_modules\@prisma\engines\query-engine-windows.exe)
Migration Engine     : migration-engine-cli cf0680a1bfe8d5e743dc659cc7f08009f9587d58 (at node_modules\@prisma\engines\migration-engine-windows.exe)
Introspection Engine : introspection-core cf0680a1bfe8d5e743dc659cc7f08009f9587d58 (at node_modules\@prisma\engines\introspection-engine-windows.exe)
Format Binary        : prisma-fmt cf0680a1bfe8d5e743dc659cc7f08009f9587d58 (at node_modules\@prisma\engines\prisma-fmt-windows.exe)
Studio               : 0.322.0
Preview Features     : microsoftSqlServer
```

**Comment by Alwigi** [comment]
I have realised that the same error occurs for both introspect and generate so it appears to be a general connection issue. Despite my port issue described below, the connection problem still persists...

I discovered that my Sql port was incorrect, the port displayed in the SQL Configuration Manager is not necessarily correct (for me it showed 1433)! This seems especially true when there are multiple sql instances running on the same machine. 
To find the correct Sql TCP/IP port use this command:

    USE master
    GO
    xp_readerrorlog 0, 1, "Server is listening on", "any", NULL, NULL, "asc"
    GO

For me the port was 50119, I have now verified that I can connect to the instance using TCP/IP:

    sqlcmd -S tcp:MyPc\SQLSERVER2017STD,50119 -U sa -P 1234

So I'm certain that I have the correct server details now but when using prisma with that config:

    DATABASE_URL="sqlserver://MyPc\SQLSERVER2017STD:50119;initial catalog=MyDb;user=sa;password=1234"

I still get the error "Could not resolve SQL browser instance SQLSERVER2017STD"
